crates/shared/src/arguments.rs

```diff
@@ -236,6 +236,31 @@ pub struct Arguments {
     /// Override address of the balancer vault contract.
     #[clap(long, env)]
     pub balancer_v2_vault_address: Option<H160>,
+
+    /// The amount of time in seconds a classification of a token into good or
```

Comment:
nit: comment is now slightly outdated since we switched to a human readable time format.